tests/test_add_model.py

```diff
@@ -0,0 +1,423 @@
+from unittest.mock import MagicMock, Mock, patch
+
+import pytest
+from flask_restful import fields
+
+from flask_restful_swagger import swagger
+
+###############################################################################
```

Comment:
This may be a bit better served, by creating a separate fixture file and importing all your setup data from there.  That way you won't have to comment out a section of this file.  Potentially easier to read, what are you thoughts?

Reply:
Yeah I think separating the test data into its own file will look a lot better.  I will make the changes and we can see how it looks after.

Reply:
Setup for fixtures moved to new file `tests/fixtures_add_model.py`